src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Color.txt

```diff
@@ -60,3 +60,12 @@ Blank()
 
 >> ColorValue("")
 Blank()
+
+>> Color.Red
+RGBA(255,0,0,1)
+
+>> Color.Black
+RGBA(0,0,0,1)
+
+>> Color.ForestGreen
+RGBA(34,139,34,1)
```

Comment:
Should we add test cases to ColorFade.txt and ColorFadeT.txt? Something like:
`ColorFade(Color.Red, 100%)`
`ColorFade(Color.Black, 100%)`
`ColorFade(Color.ForestGreen, 100%)`

Also:
`ColorFade([Color.Red, Color.Black, Color.ForestGreen], 100%)`

Reply:
Added these tests. thanks.